### DISCUSSION TITLE
feature request: cursor tracker for brush tip rotation

### DISCUSSION THREAD

**Discussion by rexelbartolome** [wojtryb/Shortcut-Composer#57] [Help me] [answered]
I'd like to have a feature that uses the cursor tracker to control the brush rotation. I think it's already quite easy to implement though I have no knowledge with scripting to actually do it. If one can explain the steps on how to implement it on my own, that would be great :)

**Comment by wojtryb** [comment]
Hi, I'd very much like to have that too.

Brush tip rotation is currently not exposed to Python (so I can't control it with an add-on). There is a workaround, but it's buggy and would break when krita 5.2 is released, so I don't want to use that.

Rotation is not exposed to add-ons due to technical limitations. One of the main features of Septembers' Krita 5.2 will be a big rework of how brush settings are programmed, which won't change how they work, but should make things like the one I talk about significantly easier (though at least for now - nobody is working on that).

My plugin is fully ready for that to happen, so once it does, it becomes a 15min task, which I'll be happy to do right away :)

**Comment by rexelbartolome** [comment]
cool! thats great to hear :)

**Comment by wojtryb** [comment, marked as answer]
This functionality is done in [1.5.0](https://github.com/wojtryb/Shortcut-Composer/releases/tag/v1.5.0)
